### PR TITLE
Journal week 3.md

### PR DESCRIPTION
**Pull request by nickycadavillo**

### CODE REVIEW

**Comment by ericksonk** [comment]
Sounds like you have a strong understanding of CSS. I need to try out time-boxing to better utilize my time spent on assignments as well. 

Good explanation of Grid & Box-sizing properties. :+1:

**Comment by karjac14** [comment]
It seems you enjoyed CSS last week and really learned a lot from it. I like the way you summarize about the topics we covered last week. I agree we really need that time boxing. :+1:

**Comment by boshnivolo** [comment]
I found this really helpful! Especially the part about how to structure the grid. :+1:

File: tutorial-week-3.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+## Grids
```
Comment (kellymurray) [inline tutorial-week-3.md]: You only need to choose one small thing to teach in this tutorial. :)

**Comment by boshnivolo** [comment]
I liked your journal! I also really enjoyed our little break from JavaScript. :+1:

**Comment by rosalsm** [comment]
This is great Nicky :+1:

**Comment by rosalsm** [comment]
We are all learning about tomatoes.:+1:

**Pull request merged**